org.uqbar.project.wollok.tests/src/org/uqbar/project/wollok/tests/interpreter/NumberTestCase.xtend

```diff
@@ -248,4 +248,13 @@ class NumberTestCase extends AbstractWollokInterpreterTestCase {
 		'''.interpretPropagatingErrors
 	}	
 
+	@Test
+	def void printString() {
+		'''program a {
+			assert.equals("4", 4.printString())
+			assert.equals("4.1", (4.1).printString())
+		}
+		'''.interpretPropagatingErrors
```

Comment:
Oh crap! I will add .test() because in another branch I have to modify all the tests also. I hope there are no conflicts.